tests/conftest.py

```diff
@@ -132,13 +132,17 @@ def schemas_mock(requests_mock: Mocker, schema_url):
     This allows to run "schema import schema afvalwegingen".
     """
     # `requests_mock` is a fixture from the requests_mock package
-    afvalwegingen_json = HERE / "files" / "afvalwegingen.json"
-    requests_mock.get(
-        f"{schema_url}index.json", json={"afvalwegingen": "afvalwegingen/afvalwegingen"}
-    )
-    with open(afvalwegingen_json, "rb") as fh:
+    AFVALWEGINGEN_JSON = HERE / "files" / "afvalwegingen_sep_table.json"
+    CLUSTERS_JSON = HERE / "files" / "afvalwegingen_clusters-table.json"
+    requests_mock.get(f"{schema_url}index.json", json={"afvalwegingen": "afvalwegingen"})
+    with open(AFVALWEGINGEN_JSON, "rb") as fh:
         requests_mock.get(
-            f"{schema_url}afvalwegingen/afvalwegingen",
+            f"{schema_url}afvalwegingen/dataset",
+            content=fh.read(),
+        )
+    with open(CLUSTERS_JSON, "rb") as fh:
```

Comment:
See issue as in https://github.com/Amsterdam/schema-tools/pull/164/files#r661767350